debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgisGeometry.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.postgresql;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+
+public class PostgisGeometry {
```

Comment:
Would not it make sense to have a common base class or interface with MySQL geometry?

Reply:
There's not much actual shared code...